axelor-base/src/main/resources/views/ProductCategory.xml

```diff
@@ -62,7 +62,8 @@
 	</action-group>
 	
 	<action-attrs name="action-attrs-product-category-form-set-hidden">
-		<attribute name="hidden" for="sequence" expr="eval: __config__.app.getApp('base').getProductSequenceTypeSelect() != 'Per product category' || __repo__(ProductCategory).all().filter('self.parentProductCategory.id = ?', id).count() > 0"/>
+		<attribute name="hidden" for="sequence" expr="eval: __config__.app.getApp('base').getProductSequenceTypeSelect() != 'Per product category' 
+			|| id ? __repo__(ProductCategory).all().filter('self.parentProductCategory.id = ?', id).count() > 0 : false"/>
```

Comment:
can be replaced by `(id && __repo__(ProductCategory).all().filter('self.parentProductCategory.id = ?', id).count() > 0)`